courses/machine_learning/deepdive2/time_series_prediction/labs/2_feature_engineering.ipynb

```diff
@@ -0,0 +1,1298 @@
+{
```

Comment:
<blockquote>Using this approach, we can use a deep neural network to model or any more "traditional" ML model.</blockquote><p>Using this approach, we can use a deep neural network or a more "traditional" ML model to train.</p><p> </p><blockquote>This approach has a nice benefit that is typically requires less feature engineering. Although, training sequentially models typically takes longer.</blockquote><p>This approach has a nice benefit that it typically requires less feature engineering. Although, training sequential models typically takes longer.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>